packages/desktop-client/src/components/budget/MobileBudgetTable.js

```diff
@@ -815,11 +894,41 @@ export function BudgetTable(props) {
   let currentMonth = monthUtils.currentMonth();
   let format = useFormat();
 
+  const toggleMobileDisplayPref = useSelector(state => {
+    return (
+      (state.prefs.local && state.prefs.local.toggleMobileDisplayPref) || true
```

Comment:
The `MobileDisplay` in `toggleMobileDisplayPref` is pretty broad and I'm also not sure I follow the "toggle" part. 

If we're directly reading this pref to decide which column to show, perhaps `mobileShowBudgetedColPref`? Then the `useState` variable could match very closely -- `showBudgetedCol`, for instance. 

Alternatively, something like `mobileBudgetFirstColPref` set to a string value `budgeted` / `spent` might be extra easy to follow through the code. 🤔